src/commands/confess.js

```diff
@@ -0,0 +1,150 @@
+const { SlashCommandBuilder } = require('@discordjs/builders')
+const { MessageEmbed } = require('discord.js')
+
+const ADMIN_CHANNEL_ID = 'ADMIN CHANNEL ID HERE (in a string)'
+const CONFESSION_CHANNEL_ID = 'CONFESSION CHANNEL ID HERE (in a string)'
```

Comment:
Can you configure these variables to be set in the .env  ?

Reply:
Should be good to go now